common/persistence/nosql/nosqlplugin/cassandra/workflow_parsing_utils.go

```diff
@@ -295,6 +295,8 @@ func parseActivityInfo(
 			info.Attempt = int32(v.(int))
 		case "task_list":
 			info.TaskList = v.(string)
+		case "task_list_kind":
```

Comment:
Does Cassandra initialize the new field to 0 or `null`?

Reply:
I think it leaves it as null and that it's not present within the Map. It's the same outcome either way though, with a default value of `NORMAL`. The way this field is added matches all the other ones.